tests/testutil.h

```diff
@@ -29,6 +29,8 @@
 #ifndef _TESTUTIL_H
 #define _TESTUTIL_H 1
 
+#include "config.h"
```

Comment:
In commit f2de47962860027889b773da05e876ea0e3d24f9 you keep `#include "config.h"` even in files that also include `testutil.h` and you provide a good explanation why that's a good idea. So is it really necessary to add this line here?

Reply:
I suspect I was in two minds about what you might prefer and ended up muddled :)

That said, `testutil.h` itself includes headers that could be affected by e.g. `AC_SYS_LARGEFILE` (`stdio.h`), so it makes sense from a perspective of ensuring all includes of standard headers are prefixed. But I can change it if you want as well.